apps/meteor/app/authorization/server/functions/upsertPermissions.ts

```diff
@@ -225,6 +225,7 @@ export const upsertPermissions = async (): Promise<void> => {
 		{ _id: 'view-import-operations', roles: ['admin'] },
 		{ _id: 'clear-oembed-cache', roles: ['admin'] },
 		{ _id: 'videoconf-ring-users', roles: ['admin', 'owner', 'moderator', 'user'] },
+		{ _id: 'bypass-time-limit-edit-and-delete', roles: ['bot'] },
```

Comment:
```suggestion
		{ _id: 'bypass-time-limit-edit-and-delete', roles: ['admin'] },
```
Maybe this permission should be assigned only to admins by default so that no unexpected behavior is introduced to users when updating to the next rc version.